app/lib/pdf_filler/id39r_pdf.rb

```diff
@@ -40,6 +43,18 @@ def hash_for_pdf
       answers
     end
 
+    def formatted_display_name
+      if @intake.filing_status_mfj?
+        if @intake.primary.last_name == @intake.spouse.last_name
```

Comment:
[super dust] i dont think this will be a likely scenario, but what if for some reason they had different capitalizations? Should we downcase & match?

Reply:
🤔 it’s not a bad idea, i pushed up a fix for this.